tests/app/service/test_rest.py

```diff
@@ -2622,7 +2656,7 @@ def test_get_all_notifications_for_service_includes_template_redacted(
 
 # TODO: check whether all hidden templates are also precompiled letters
 # def test_get_all_notifications_for_service_includes_template_hidden(admin_request, sample_service):
-#     letter_template = create_template(sample_service, template_type=LETTER_TYPE)
+#     letter_template = create_template(sample_service, template_type=TemplateType.LETTER)
```

Comment:
This is likely a test we could remove in the future given the removal of letter processing.